mxcube3/__init__.py

```diff
@@ -82,7 +82,7 @@ def complete_initialization(app):
         app.session = app.beamline.getObjectByRole("session")
         app.collect = app.beamline.getObjectByRole("collect")
         app.diffractometer = app.beamline.getObjectByRole("diffractometer")
-        if not hasattr(app.diffractometer, 'centring_motors_list'):
+        if getattr(app.diffractometer, 'centring_motors_list') is None:
```

Comment:
Nothing wrong with pep8 and hasattr... but it was not working as intended:
 ```hasattr(app.diffractometer, 'centring_motors_list')``` and ```hasattr(app.diffractometer, 'whateveryouputhere')``` were returning ```True``` in both cases.
In addition, the ```getattr``` was returning ```None``` for the mockup, no exception raising

I think I am missing something...